.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
   build:
     concurrency:
-      group: ${{ github.workflow }}-${{ strategy.job-index }}-${{ github.event_name == 'pull_request' && github.ref || github.sha }}
+      group: ${{ github.job }}-${{ strategy.job-index }}-${{ github.event_name == 'pull_request' && github.ref || github.sha }}
```

Comment:
should use `github.job` since `github.workflow` represents the workflow filename